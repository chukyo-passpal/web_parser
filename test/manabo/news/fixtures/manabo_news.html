<div class="panel panel-default sp-margin-bottom-none sp-border-bottom-none sp-border-top-none">
    <div class="panel-heading">お知らせ</div>
    <div class="sp-margin-bottom-md sp-padding-sm">
        <table class="table-no-border table-info">
            <tbody>
                <tr>
                    <td class="padding-md">現在システムからのお知らせはありません</td>
                </tr>
            </tbody>
        </table>
    </div><!--table-responsive-->
</div><!--panel-->
