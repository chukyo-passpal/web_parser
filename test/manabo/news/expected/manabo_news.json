{
  "message": "現在システムからのお知らせはありません",
  "items": []
}